README.md

```diff
@@ -1,19 +1,147 @@
-# Postal on kubernetes
+# postal
 
-A docker-image and helm-chart to deploy [Postal](https://postal.atech.media/) on kubernetes.
+A helm-chart to deploy [Postal](https://github.com/postalserver/postal) on kubernetes.
 
-Please refer to the individual documentation for the docker-image and the helm-chart:
+## Introduction
 
-- [docker-image](docker/README.md)
-- [helm-chart](helm/postal/README.md)
+This chart bootstraps a deployment of Postal, MariaDB and RabbitMQ on a
+[Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
-Getting the Docker image and helm chart:
+## Prerequisites
 
-- The docker-image is available on Docker Hub: [linkyard/postal](https://hub.docker.com/r/linkyard/postal/)
-- The helm chart is available at [charts.linkyard.ch](https://charts.linkyard.ch)
+- Kubernetes 1.9
+- PV provisioner support in the underlying infrastructure
 
-## Roadmap
+## Optional prerequisites
 
-- Contribute our little fixes to upstream
-- Add the ability to start individual Portal processes for easier HA setups and scaling
-- Add meaningful readiness and liveness probes
+- An ingress controller
+- A functioning [cert-manager](https://github.com/jetstack/cert-manager) for certificate management
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`, add the helm charts repository:
+
+```console
+clone this repository
+```
+
+Setup an ingress-controller
+
+```console
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+```
+
+```console
+helm install my-ingress ingress-nginx/ingress-nginx `
+     --namespace ingress `
+     --create-namespace `
+     --set controller.replicaCount=2 `
+     --set controller.nodeSelector."beta\.kubernetes\.io/os"=linux `
+     --set defaultBackend.nodeSelector."beta\.kubernetes\.io/os"=linux
+```
+
+Update your domain info using the IP from ingress controller to add to your @/* (A) dns record
```

Comment:
We should explain how to find the IP of the ingress controller. 

Also, the language here seems to imply that either an @ or * record would be sufficient. Is that the case or do you need both (my understanding was need both)?